Added a start screen

vincetiu8 opened the pull request:
This commit separates the actual 'level' from the game, making future updates later to add levels and a settings screen easier. It also adds a start screen.

vincetiu8 committed:
Added a start screen

This commit separates the actual 'level' from the game, making future updates later to add levels and a settings screen easier. It also adds a start screen.

Graphcalibur reviewed (changes requested):
Works mostly fine, but pressing R on the start screen crashes the game.

vincetiu8 committed:
Fix bug

This commit fixes a bug that causes the start screen to crash if the key 'r' is pressed.
